src/vs/workbench/contrib/scm/browser/media/scm.css

```diff
@@ -173,6 +173,10 @@
 	background: transparent !important;
 }
 
+.scm-view .monaco-list .monaco-list-row.force-cursor-default {
+	cursor: default !important;
```

Comment:
Can we find another way which does not require `!important`?

Reply:
@joaomoreno, I can certainly look into that. 
I went this way as there was already a similar pattern to disable hovering.

Reply:
Pushed an update to remove `!important`.